src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/prison/PrisonApiClient.kt

```diff
@@ -19,7 +19,7 @@ class PrisonApiClient(@Qualifier("oauthPrisonClient") val prisonerSearchApiWebCl
   fun getBooking(bookingId: Long): Booking? {
     val booking = prisonerSearchApiWebClient
       .get()
-      .uri("/bookings/$bookingId")
+      .uri("/bookings/{bookingId}", bookingId)
```

Comment:
prevent garbage leak, it registers a micrometer tag for each unique url passed to this method